src/lib/inputs/Prefix.svelte

```diff
@@ -75,12 +76,20 @@
 
 <div class="input-container">
   {#if label}
-    {#if labelIn}
-      <label class="label-in" for={$$restProps.id}>{$$restProps.id}</label>
-    {:else}
-      <label class="label" for={$$restProps.id}>{$$restProps.id}</label>
+        {#if labelIn}
+            <label
+                class="label-in"
+                for={$$restProps.id}
+                style="color: {labelColor}">{$$restProps.id}</label
+            >
```

Comment:
what does 'labelIn' mean, I'm not sure it's going to be super intuitive to a user

Reply:
labelIn is to make the label appear, at first, inside the input. On focus the label floats to outside the input.

Hmmm, I wonder if there's a more semantically correct way to name this?

Reply:
I will play around with this idea. My aim is ease of use and having a user type 20 characters does not fit that bill lol.